src/lib/components/LavaLampBG.svelte

```diff
@@ -0,0 +1,164 @@
+<script lang="ts">
+	import { onMount } from 'svelte';
+
+	export let orbCount = 25;
+	export let baseRadius = 50;
+	export let addedRadius = 50;
+	export let baseVel = 0.5;
+	export let addedVel = 0.5;
+
+	export let alphaThreshold = 200;
+
+	let canvasWidth: number;
+	let canvasHeight: number;
+
+	let skipInitialTesting = false;
+	let disabledAnimation = false;
+
+	let canvasElement: HTMLCanvasElement;
+	let fakeCanvas: HTMLCanvasElement;
+
+	let context: CanvasRenderingContext2D | null;
+	let finalContext: CanvasRenderingContext2D | null;
+	$: if (canvasElement) {
```

Comment:
you can remove this and silly boilerplate and height logic through [svelte-canvas](https://www.npmjs.com/package/svelte-canvas)

Reply:
I see no other advantages to this, and there is no point in refactoring working code using extra dependencies that work very similarly to the existing code IMO.